### DISCUSSION TITLE
Label text modification

### DISCUSSION THREAD

**Discussion by KorecGroup-AliJafri** [zouyaoji/vue-cesium#489] [Q&A] [answered]
Hi! Looking at the vc-drawing options and tools available, would it be possible to amend the label text appearing on a pinDraw button call? The amend would happen by firing an input field and entering in text prompt input field, this would then amend the label text?

**Comment by zouyaoji** [comment, marked as answer]
Hello, not possible. I suggest you first draw a default label in the viewer, and then modify it in a pop-up window.

**Reply by KorecGroup-AliJafri** [reply]
Thanks!
